cluster/gce/trusty/configure-helper.sh

```diff
@@ -532,14 +523,9 @@ start_kube_scheduler() {
   timeout 30 docker load -i "${kube_home}/kube-docker-files/kube-scheduler.tar"
 
   # Calculate variables and set them in the manifest.
-  params=""
-  log_level="--v=2"
+  params="--v=2 ${SCHEDULER_TEST_ARGS:-}"
   if [ -n "${SCHEDULER_TEST_LOG_LEVEL:-}" ]; then
-    log_level="${SCHEDULER_TEST_LOG_LEVEL}"
-  fi
-  params="${params} ${log_level}"
-  if [ -n "${SCHEDULER_TEST_ARGS:-}" ]; then
-    params="${params} ${SCHEDULER_TEST_ARGS}"
+    params="${SCHEDULER_TEST_LOG_LEVEL} ${SCHEDULER_TEST_ARGS:-}"
```

Comment:
Same as above, the "if" block reassign value to params. Let me change it to a more readable style.